classes/form/upload_bookings_form.php

```diff
@@ -59,11 +60,14 @@ public function definition() {
         $mform->setType('csvfile', PARAM_INT);
         $mform->addRule('csvfile', get_string('required'), 'required', null, 'client');
 
+        // Allowed statuses minus the '' one, as we have additional help text to explain '' status specifically.
+        $allowedstatuses = array_filter(booking_manager::get_allowed_session_statuses(), fn($status) => !empty($status));
```

Comment:
```suggestion
        $allowedstatuses = array_filter(booking_manager::get_allowed_session_statuses());
```
By default it strips out empty/falsey entries, so you shouldn't need to supply any custom function to handle it.